WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -29,6 +29,9 @@ class CompetitionsController < ApplicationController
 
   def new
     @competition = Competition.new
+    if current_user.any_kind_of_delegate?
+      @competition.delegates |= [current_user]
```

Comment:
Aha! Don't think I've ever encountered `Array.|` before!

I'm used to seeing `@competition.delegates << current_user` to do this.